config/settings.yml

```
@@ -8,3 +8,8 @@ url:
   rails_screencasts_url: "https://screencasts.railstutorial.org/"
   learn_enough_url: "https://www.learnenough.com/"
   author_url: "https://www.michaelhartl.com/"
+validate:
+  name_max_length: 50
+  pass_min_length: 6
+  email_max_length: 255
```

```suggestion
validate:
  length:
    length_50: 50
    length_6: 6
    length_255: 255
```
ri để tái sử dụng dc cao em nhé
chứ em thêm tên field vào thì ko thể tái sử dụng dc. mà đọc vô cũng ko bjk giá trị em đang muốn setting là bao nhiu cả